app/models/manageiq/providers/amazon/manager_mixin.rb

```diff
@@ -76,7 +76,7 @@ def verify_credentials(args)
       # Pull out the password from the database if a provider ID is available
       secret_access_key ||= find(args["id"]).authentication_password('default')
 
-      !!raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, validate = true, :assume_role => service_account)
+      !!raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, validate = true, :assume_role => service_account) # rubocop:disable Lint/UselessAssignment (see #588)
```

Comment:
Lets pull `validate = true` up instead of adding rubocop:disable comments
```
validate = true

raw_connect(..., validate)
```

I just don't want to pass `true` as an argument since that's not very descriptive

Reply:
Or we can pull the validate logic out of `raw_connect` and instead do in `verify_credentials`:
```
connection = raw_connect(access_key, secret_access_key, :EC2, region, proxy_uri, :assume_role => service_account)
!!validate_connection(connection)
```

I actually like that better, raw_connect just returns the connection.  It is up to verify_credentials to...verify the credentials

Reply:
Ok, updated.